scripts/db_migrator.py

```diff
@@ -39,7 +39,7 @@ def __init__(self):
         self.TABLE_NAME      = 'VERSIONS'
         self.TABLE_KEY       = 'DATABASE'
         self.TABLE_FIELD     = 'VERSION'
-        self.configDB        = ConfigDBConnector()
+        self.configDB        = ConfigDBConnector(**args)
```

Comment:
Since these are keyword args, it's more clear to follow the Python convention of naming it `kwargs`. Also, it is a bit unclear here what this variable can/should contain.

Reply:
I'll edit the variable name.

I'm using `args` to hold the path of the unix socket (if one is given), this was the style I saw used in sonic-cfggen to do the same thing.

Reply:
Understood. I just think it could be made clearer (maybe just comments) as to what arguments can be passed to the constructor of DBMigrator.

Reply:
I agree we should have a more meaningful name for argument. Since this is used for db_connector, you can simply name it db_args or something alike.

Reply:
re-structured it a little bit to make the constructor clearer